core/src/main/java/com/linecorp/armeria/common/logging/LoggingDecoratorBuilder.java

```diff
@@ -420,12 +502,68 @@ public LoggingDecoratorBuilder responseCauseSanitizer(
         return responseCauseSanitizer;
     }
 
+    /**
+     * Sets the {@link Predicate} for evaluating to log response cause or not.
+     * You can prevent from logging the response cause by returning {@code true}
+     * in the {@link Predicate}. If unset, will log a response cause.
+     */
+    @UnstableApi
+    public LoggingDecoratorBuilder responseCauseFilter(Predicate<Throwable> responseCauseFilter) {
+        this.responseCauseFilter = requireNonNull(responseCauseFilter, "responseCauseFilter");
+        return this;
+    }
+
+    /**
+     * Returns the {@link Predicate} to evaluate if logging response cause or not.
+     */
+    @UnstableApi
+    protected final Predicate<Throwable> responseCauseFilter() {
+        return responseCauseFilter;
+    }
+
+    /**
+     * Sets the {@link LogFormatter} to converts a {@link RequestOnlyLog} or {@link RequestLog}
+     * into log message. If unset, will use {@link TextLogFormatter}.
```

Comment:
since `TextLogFormatter` is package private

```suggestion
     * Sets the {@link LogFormatter} which converts a {@link RequestOnlyLog} or {@link RequestLog}
     * into a log message. By default {@link LogFormatter#ofText()} will be used.
     *
```